pygluu/containerlib/document/rclone.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import os
+
+from ..utils import exec_cmd
+
+logger = logging.getLogger(__name__)
+
+
+class RClone:
+    def __init__(self, url, username, password):
+        self.url = "localhost:8080/repository/default"
+        self.username = username
+        self.password = password
+
+    def configure(self):
+        conf_file = os.path.expanduser("~/.config/rclone/rclone.conf")
+        if os.path.isfile(conf_file):
+            return
+
+        cmd = "rclone config create jackrabbit webdav vendor other pass {self.password} user admin url {self.url}"
+        _, err, code = exec_cmd(cmd)
+
+        if code != 0:
+            errors = err.decode().splitlines()
+            logger.warning("Unable to create webdav config; reason={errors}")
```

Comment:
The `{errors}` will not be pre-populated correctly. Please take a look at string formatting in Python2.